src/ontology/edits/oeo-physical.omn

```diff
@@ -4696,6 +4721,92 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/950",
         OEO_00010210
     
     
+Class: OEO_00010216
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A power-to-gas process is an energy transformation that converts electrical energy to chemical energy by synthesising portions of matter into a gaseous synthetic fuel.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "P2G",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "PtG",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "power to gas",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/940
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/954",
+        rdfs:label "power-to-gas process"@en
+    
+    SubClassOf: 
+        OEO_00020003,
+        (OEO_00000532 some OEO_00000139)
+         and (OEO_00000532 some OEO_00000331),
+        (OEO_00000533 some OEO_00000007)
+         and (OEO_00000533 some OEO_00010155)
+    
+    
+Class: OEO_00010217
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A power-to-methane process is a power-to-gas process that has electrical energy, water and carbon dioxide as physical input and synthetic methane carrying chemical energy as physical output. It consists of two sub processes: an electrolysis process and a methanation.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/940
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/954",
+        rdfs:label "power-to-methane process"@en
+    
+    SubClassOf: 
+        OEO_00010216,
+        (OEO_00000532 some OEO_00000006)
+         and (OEO_00000532 some OEO_00000139)
+         and (OEO_00000532 some OEO_00000441),
+        (OEO_00000533 some OEO_00000007)
+         and (OEO_00000533 some OEO_00010018),
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010219,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010220
+    
+    
+Class: OEO_00010218
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Oxygen is a portion of matter with the chemical formula O2. It has a gaseous normal state of matter.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "O2",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/940
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/954",
+        rdfs:label "oxygen"@en
+    
+    SubClassOf: 
+        OEO_00000331,
+        OEO_00000529 value OEO_00000182
+    
+    
+Class: OEO_00010219
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A water electrolysis process is energy transformation that uses an electric current to decompose water into hydrogen and oxygen gas. Electric energy is converted into chemical energy (and waste heat).",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A water electrolysis process is an energy transformation that uses an electric current to decompose water into hydrogen and oxygen gas. Electric energy is converted into chemical energy (and waste heat).",
```

Reply:
change `electric energy` to `electrical energy`

Reply:
We should also change the definition of `water electrolyser` because you used this definition.

Reply:
As I already merged to the PR, I'll fix this in #959